src/Rating.php

```diff
@@ -115,7 +115,13 @@ public function withStyles(array $styles)
     {
         $build = [];
         foreach (static::$defaultStyles as $key => $defaultValue) {
-            $build[$key] = Arr::get($styles, $key, $defaultValue);
+            // In Laravel 6.5 array_get function not exist 
+            if (function_exists('array_get')) {
```

Comment:
This is conditional is not needed. `Arr::get` should be enough. It has been around since at least 5.0 and is not the only way.